iocore/net/I_NetVConnection.h

```diff
@@ -40,6 +40,13 @@
 #define SSL_EVENT_SERVER 0
 #define SSL_EVENT_CLIENT 1
 
+// Indicator the context for a NetVConnection
+typedef enum {
+  Net_VConnection_UNSET,
+  Net_VConnection_C2P, // Client <--> ATS, Client-Side
+  Net_VConnection_P2S, // ATS <--> Server, Server-Side
```

Comment:
Instead of C2P and P2S, NET_CONNECTION_IN,  NET_CONNECTION_OUT, and  NET_CONNECTION_UNSET would be better.  We normally uppercase enums.